api/models/participant.py

```diff
@@ -2,10 +2,12 @@
 from django.conf import settings
 from django.core import mail
 from django.db import models
+from django.utils.translation import ugettext_lazy as _
```

Comment:
Calling a function "_" is the most horryfing thing anyone has ever done. I realize it has now become some kind of convention, but I am just not strong enough to approve this without you saying there is no other way to do this.